dashboard/controller/classrooms/deleteClassroom.js

```diff
@@ -14,11 +14,11 @@ module.exports = function deleteClassroom(req, res) {
 
 					// send to classrooms page
 					req.flash('success', {
-						msg: common.l10n.get('ClassroomDeleted', {name: name})
+						msg: common.l10n.get('ClassroomDeleted', { name: name })
 					});
```

Comment:
If changes in the code of this file does not affects the Assignment Feature, then reset the changes in this file.